### PR TITLE
Fix Draw - Page Layout cannot be open from top tool sidebar icon

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
Change-Id: If9a57484a8335034e3007f47513cd4827c8decbc

* Resolves: #6405 
* Target version: master 

- Fixed sidebar toogle functionality in odg (draw) file

### CODE REVIEW

**Review by eszkadev — CHANGES_REQUESTED** [review]
this cannot be done because then it will not be possible to switch back from navigator I think

File: browser/src/control/Control.Sidebar.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ L.Control.Sidebar = L.Control.extend({
 	},
 
 	changeDeck: function(unoCommand) {
-		if (unoCommand !== null)
+		if (unoCommand !== null&& unoCommand !== '.uno:ModifyPage')
```
Comment (codewithvk) [inline browser/src/control/Control.Sidebar.js:155]: I don't think we should add a condition here. Instead for the top right button, we can add fix here: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.TopToolbar.js#L242

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.Sidebar.js:155]: yes I forget to remove this . thanks :)

**Review by eszkadev — APPROVED** [review]
looks good :)

**Pull request merged**